modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/connectors/JMSClientTest.java

```diff
@@ -51,22 +51,6 @@ public void testJMSClientConnectorWithoutValidInitialContextFactory() throws Bal
         BLangFunctions.invoke(bLangProgram, "jmsClientConnectorTest");
     }
 
-    @Test(description = "Test for jms client connector without valid message",
```

Comment:
@amjadhifthikar What is the reason for removing these test cases? If these are no longer valid, can you add test cases to cover your fixes?